src/daemon/daemon.cpp

```diff
@@ -98,6 +98,14 @@ bool Daemon::activate(const InterfaceConfig& config) {
       return false;
     }
   }
+  if (supportIPUtils()) {
+    if (!iputils()->addInterfaceIPs(config)) {
```

Comment:
The clean-up for this is just to delete the interface. That is done with wgutils. I think it's okay to not try and clean this up if it fails. We return false which means we know that activation failed. Thoughts?

Reply:
We could change the whole structure of activate to try removing the interface if activation fails and if removing the interface fails, we could bail out altogether with a qFatal?

Reply:
In an ideal world I'm thinking that wgutils and iputls can be added seperately and we don't need to have one to have the other. But maybe that's not a thing to really worry about.